channels/urbdrc/client/libusb/libusb_udevice.c

```diff
@@ -1763,23 +1776,28 @@ size_t udev_new_by_id(URBDRC_PLUGIN* urbdrc, libusb_context* ctx, UINT16 idVendo
 
 	for (i = 0; i < total_device; i++)
 	{
-		LIBUSB_DEVICE_DESCRIPTOR* descriptor = udev_new_descript(urbdrc, libusb_list[i]);
+		current_dev = libusb_list[i];
```

Comment:
variable must be defined here instead of parent scope (VS2010 does not like mixing assignment and declaration later on)